tools/hermetic_grunt/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# To build and run the hermetic grunt:
+# docker build -f tools/hermetic_grunt/Dockerfile -t uproxy-grunt . && \
+# docker run --rm -v $(pwd):/root/repository_root -it uproxy-grunt
+#
+# You have to run from the repository root. The output is written to the out/
+# directory.
+
+FROM ubuntu
```

Comment:
Let's specify a version. Ideally, all our Docker images would use the same base image. Right now, `ubuntu:xenial` is the latest but our other Docker images use `ubuntu:trusty`:
https://github.com/uProxy/uproxy-docker/blob/master/sshd/Dockerfile